src/lib.rs

```diff
@@ -3176,7 +3176,7 @@ impl std::fmt::Debug for Stats {
             self.lost,
             self.rtt,
             self.cwnd,
-            self.delivery_rate
+            self.delivery_rate as u64
```

Comment:
Maybe we should just make the field `u64`? Not quite sure how useuful it is to keep it as float since we just convert it to `u64` in every place where we use it (e.g. FFI does the same to mimic what `TCP_INFO` does).

Reply:
Good idea. I opened #422 including this diff.